protocol/avdl/keybase1/tlf.avdl

```diff
@@ -18,5 +18,5 @@ protocol tlf {
   // The tlfName can be either complete or partial. If the user's username is
   // not present in tlfName, it gets appended automatically. The canonical TLF
   // name is returned.
-  CanonicalTLFNameAndIDWithBreaks completeAndCanonicalizeTlfName(TLFQuery query);
+  CanonicalTLFNameAndIDWithBreaks completeAndCanonicalizeTlfName(TLFQuery query, bool isPublic);
```

Comment:
We have separate calls for public/private TLFs (`GetTLFCryptKeys` vs `GetPublicCanonicalTLFNameAndID`), and those accept a `TLFQuery`. I thought it would be a bit redundant?